transform-plugins/src/main/java/co/cask/hydrator/plugin/CSVParser.java

```diff
@@ -272,27 +273,23 @@ private StructuredRecord createStructuredRecord(@Nullable CSVRecord record, Stru
     @Description("Specifies the schema that has to be output.")
     private final String schema;
 
-    public Config(String format, Character delimiter, String field, String schema) {
-      this.format = format;
+    public Config(@Nullable String format, @Nullable Character delimiter, String field, String schema) {
+      this.format = format == null ? "DEFAULT" : format;
```

Comment:
are you trying to default format to "DEFAULT"? If so, this is not the way to do it. You have to do it in a no-args constructor. This will throw NPE if you actually try to use it.